docs/UserGuide.md

```diff
@@ -97,6 +97,13 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS g/GENDER`
 - Adds gender to a person in the contact list. The genders accepted by the contact list are: `M`/`m`/`Male`/`male`
 for male, `F`/`f`/`Female`/`female` for female.
 
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** The user cannot add a new person with an already existing name in the application.<br><br>
```

Comment:
I think it'll be better not to use the term 'user' to refer to the individual reading the guide to standardize with the rest of the guide.
```suggestion
:information_source: **Note:** A new person that has the same name as an existing person in the application cannot be added to the application.<br><br>
```